Error generating javadocs with JDK 13

Environment:
```
$ mvn --version
Apache Maven 3.6.0 (97c98ec64a1fdfee7767ce5ffb20918da4f719f3; 2018-10-24T20:41:47+02:00)
Maven home: /home/jsenko/apps/apache-maven-3.6.0
Java version: 13.0.2, vendor: Oracle Corporation, runtime: /usr/java/jdk-13.0.2
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "5.1.18-200.fc29.x86_64", arch: "amd64", family: "unix"
```
Command:
```
mvn clean install
```
Error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadocs) on project apicurio-studio-shared-beans: MavenReportException: Error while generating Javadoc: 
[ERROR] Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.
[ERROR] 
[ERROR] Command line was: /usr/java/jdk-13.0.2/bin/javadoc @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/jsenko/projects/***/repos/Apicurio/apicurio-studio.git/shared/beans/target/apidocs' dir.
[ERROR] 
```

--- a/pom.xml
+++ b/pom.xml
@@ -110,6 +110,7 @@
         <version.commons-lang3>3.9</version.commons-lang3>
         <version.io.gatling>3.3.1</version.io.gatling>
         <version.io.prometheus>0.8.1</version.io.prometheus>
+        <version.javax.annotation>1.3.2</version.javax.annotation>
         <version.javax.enterprise>2.0</version.javax.enterprise>
         <version.javax.jms.jms-api>2.0.1</version.javax.jms.jms-api>
         <version.junit>4.13</version.junit>
@@ -323,6 +324,11 @@
                 <groupId>com.sun.codemodel</groupId>
                 <artifactId>codemodel</artifactId>
                 <version>${version.com.sun.codemodel}</version>
+            </dependency>
+            <dependency>
+                <groupId>javax.annotation</groupId>
+                <artifactId>javax.annotation-api</artifactId>
+                <version>${version.javax.annotation}</version>
             </dependency>
 
             <!-- Gatling -->
@@ -711,6 +717,29 @@
                         <artifactId>maven-javadoc-plugin</artifactId>
                         <configuration>
                             <additionalparam>-Xdoclint:none</additionalparam>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>java13</id>
+            <activation>
+                <jdk>(1.8,]</jdk>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>javax.annotation</groupId>
+                    <artifactId>javax.annotation-api</artifactId>
+                </dependency>
+            </dependencies>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <configuration>
+                            <source>8</source>
                         </configuration>
                     </plugin>
                 </plugins>

